src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const arrayPopulationStr = [
+  ...document.getElementsByClassName('population'),
+].map((n) => n.textContent);
+
+const arrayPopulationNum = arrayPopulationStr.map((str) => {
+  return Number(
+    str
+      .split('')
+      .filter((element) => element !== ',')
+      .join(''),
+  );
+});
+
+const total = arrayPopulationNum.reduce(
+  (accumulator, currentValue) => accumulator + currentValue,
+  0,
+);
```

Comment:
The reduce function is correctly used to calculate the total population. The initial value of 0 is appropriately set.